pkgs/by-name/wa/waterfox/package.nix

```
@@ -0,0 +1,210 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  wrapGAppsHook3,
+  alsa-lib,
+  at-spi2-atk,
+  at-spi2-core,
+  atk,
+  cairo,
+  cups,
+  dbus-glib,
+  libnotify,
+  libdrm,
+  libxkbcommon,
+  mesa,
+  nspr,
+  nss,
+  pango,
+  systemd,
+  libpulseaudio,
+  gdk-pixbuf,
+  gtk3,
+  makeDesktopItem,
+  unzip,
+  libX11,
+  libXScrnSaver,
+  libXcomposite,
+  libXcursor,
+  libXdamage,
+  libXext,
+  libXfixes,
+  libXi,
+  libXrandr,
+  libXrender,
+  libXtst,
+  libxcb,
+  writeText,
+  patchelf,
+  makeWrapper,
+  libGL,
+  ffmpeg,
+  glib,
+  pciutils,
+}:
+
+let
+  version = "6.5.4";
+
+  desktopItem = makeDesktopItem {
+    name = "waterfox";
+    exec = "waterfox %U";
+    icon = "waterfox";
+    desktopName = "Waterfox";
+    genericName = "Web Browser";
+    categories = [
+      "Network"
+      "WebBrowser"
+    ];
+    mimeTypes = [
+      "text/html"
+      "text/xml"
+      "application/xhtml+xml"
+      "application/vnd.mozilla.xul+xml"
+      "x-scheme-handler/http"
+      "x-scheme-handler/https"
+      "x-scheme-handler/ftp"
+    ];
+  };
+
+  prefsFile = writeText "waterfox-prefs.js" ''
+    // Disable automatic updates and notifications
+    pref("app.update.auto", false);
+    pref("app.update.enabled", false);
+    pref("browser.crashReports.unsubmittedCheck.enabled", false);
+  '';
+
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "waterfox";
+  version = version;
```

```suggestion
  version = "6.5.4";
```
The other `version` is redundant

---

This is incorrect. You should use `inherit version;`.

---

You are right it was not necessarily needed before, but I also added `changelog = "https://github.com/BrowserWorks/Waterfox/releases/tag/${version}` that now depends on it. 

I will keep it this way.